lib/mock-server/pretender-config.js

```diff
@@ -47,11 +48,65 @@ const defaultPassthroughs = [
   },
 ];
 
+const defaultRouteOptions = {
+  coalesce: false,
+  timing: undefined,
+};
+
+/**
+ * Determine if the object contains a valid option.
+ *
+ * @method isOption
+ * @param {Object} option An object with one option value pair.
+ * @return {Boolean} True if option is a valid option, false otherwise.
+ * @private
+ */
+function isOption(option) {
+  if (!option || typeof option !== "object") {
+    return false;
+  }
+
+  let allOptions = Object.keys(defaultRouteOptions);
+  let optionKeys = Object.keys(option);
+  for (let i = 0; i < optionKeys.length; i++) {
+    let key = optionKeys[i];
+    if (allOptions.indexOf(key) > -1) {
+      return true;
+    }
+  }
+  return false;
+}
+
 /**
  @hide
  */
 export { defaultPassthroughs };
 
+/**
+ * Extract arguments for a route.
+ *
+ * @method extractRouteArguments
+ * @param {Array} args Of the form [options], [object, code], [function, code]
+ * [shorthand, options], [shorthand, code, options]
```

Comment:
This seems like a pretty funky format.  Maybe an example here would help make this more clear.  That's just a nice-to-have, though.

Edit, oh I see, you just moved this from elsewhere.  Ignore these couple of comments if you like.

Reply:
Yes, this code was just moved around from the original code